### PR TITLE
Created footer and burger_menu without adaptive

### PR DESCRIPTION
**Pull request by Yevhenii0536**

### CODE REVIEW

File: src/BurgerMenu/BurgerMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+//   const { Cross, Heart, ShoppingBag } = images;
+
+  return (
+    <nav className="menu">
+      <div className="menu__content">
+        <div className="top-actions">
+          <div className="top-actions__logo">
+            <a href="/">
+              <img src={logo} alt="logo" />
+            </a>
```
Comment (KuharchukDmytro) [inline src/BurgerMenu/BurgerMenu.tsx:16]: use link from react-router

File: src/Footer/Footer.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
+.footer {
+    box-sizing: border-box;
```
Comment (KuharchukDmytro) [inline src/Footer/Footer.scss:2]: this property in normalize file, u can delete it

File: src/Footer/Footer.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    border: 1px solid #E2E6E9;
+    height: 98px;
+    padding: 10px;
+    width: 100%;
+    height: 96px;
```
Comment (KuharchukDmytro) [inline src/Footer/Footer.scss:7]: height duplicating, and for such global properties better to use variables

File: src/Footer/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+export const Footer: React.FC = () => {
+  return (
+    <footer className="footer">
+      <a href="/">
+        <img src={logo} alt="a" />
+      </a>
+
+      <ul className="footer-nav">
+        <li>
+          <a className="footer-nav_link" href="/">
```
Comment (KuharchukDmytro) [inline src/Footer/Footer.tsx:15]: wrong bem usage
```suggestion
          <a className="footer-nav__link" href="/">
```

**Review by KuharchukDmytro — CHANGES_REQUESTED** [review]

**Review by lilia-obushenko — APPROVED** [review]

**Review by KuharchukDmytro — APPROVED** [review]

**Review by KuharchukDmytro — APPROVED** [review]

**Pull request merged**